commands/group_test.go

```diff
@@ -166,3 +166,303 @@ func (s *MmctlUnitTestSuite) TestTeamGroupEnableCmd() {
 		s.Len(printer.GetErrorLines(), 0)
 	})
 }
+
+func (s *MmctlUnitTestSuite) TestChannelGroupEnableCmdF() {
+	s.Run("Enable group constrains with existing team and channel", func() {
```

Comment:
Small typo here and in the cases below, should be `constraints`.

Reply:
Thank you for the feedback. This has been corrected now.